src/App.css

```diff
@@ -1,28 +1,130 @@
-.App {
+@font-face {
+  font-family: 'Blacklisted';
+  src: url('./assets/Blacklisted.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: 'Lato-regular';
+  src: url('./assets/Lato/Lato-Regular.ttf') format('truetype');
+}
+
+#root {
+  max-width: 1900px;
+  margin: 0 auto;
   text-align: center;
+  background-color: #f5f5f5;
+}
+
+main {
+  max-width: 1700px;
+  display: grid;
+  justify-content: end;
+  margin-left: 70px;
+  font-family: 'Lato-regular';
+}
+
+.header {
+  border: 2px solid indigo;
+  width: 100%;
+  background-color: indigo;
+  border-bottom-right-radius: 30px;
+}
+
+.footer {
+  background-color: indigo;
+  height: 40px;
+  display: relative;
+  border: 2px solid indigo;
+  color: white;
+  margin-bottom: 0px;
+  border-top-left-radius: 30px;
+  border-top-right-radius: 30px;
+}
+
+.title {
+  color: white;
+  margin-top: 0px;
+  font-family: 'Blacklisted'
 }
 
-.App-header {
-  background-color: #282c34;
+form{
   display: flex;
-  flex-direction: row;
-  align-items: center;
-  justify-content: space-evenly;
+  justify-content: flex-start;
+  margin-left: 25px;
+}
+
+#search-bar {
+  padding: 10px;
+  width: 500px;
+  margin-right: 3px;
+
+}
+
+#search-button {
+  background-color: #B19CD9;
+  padding: 10px;
+  border: 2px solid darkmagenta;
+  border-radius: 5px;
+}
+
+#search-button:hover,
+#nowPlayingButton:hover,
+#searchTabButton:hover {
   color: white;
-  padding: 20px;
+  background-color: darkmagenta;
+  border: 2px solid #B19CD9;
+  border-radius: 5px;
 }
 
-@media (max-width: 600px) {
-  .movie-card {
-    width: 100%;
-  }
+#nowPlayingButton {
+  background-color: #B19CD9;
+  padding: 10px;
+  border: 2px solid darkmagenta;
+  border-radius: 5px;
+  margin-right: 10px;
+}
+
+#searchTabButton {
+  background-color: #B19CD9;
```

Comment:
we reuse `background-color: #B19CD9;` quite a few times here, in the future let's try to capture repeated styles in a separate class so we can avoid repetition